sdks/java/core/src/main/java/org/apache/beam/sdk/testing/BigqueryMatcher.java

```diff
@@ -210,9 +219,9 @@ public void describeTo(Description description) {
   @Override
   public void describeMismatchSafely(PipelineResult pResult, Description description) {
     String info;
-    if (response == null || response.getRows() == null || response.getRows().isEmpty()) {
-      // invalid query response
-      info = String.format("Invalid BigQuery response: %s", Objects.toString(response));
+    if (!response.getJobComplete()) {
+      // query job not complete
+      info = String.format("Query job not complete with retry. Got response: %s", response);
```

Comment:
"The query job hasn't completed"

Could we also add the job id? Is response.toString() readable (why it was "Objects.toString(response)").